valentineezeh/Ride-My-Way
# #158412694 Configure base app

#### What does this PR do?
This PR configures the base setup for the application

#### Description of task to be completed
- set up babelrc file
- set up gitignore
- set up express server
- set up routes

#### How can this be manually tested
- on the command line type npm start to start the server.
- then input the router url into the browser to see the welcome message.

#### Relevant pivotal tracker story
[158412694](https://www.pivotaltracker.com/story/show/158412694)
